lib/inc/cpp-pcp-client/protocol/serialization.hpp

```diff
@@ -96,7 +96,7 @@ inline void serialize(const T& thing,
     } catch(const std::exception& e) {
         throw message_serialization_error { e.what() };
     } catch(...) {
-        throw message_serialization_error { "seriliazation: unexpected error "
+        throw message_serialization_error { "serialization: unexpected error "
```

Comment:
I avoided touching headers for https://github.com/puppetlabs/leatherman#limitations.
This is the only header we need translation in cpp-pcp-client. But in pxp-agent we have more cases in Configuration's template functions. 
Suggestions?